packages/react-components/components/CallToActionsBar.tsx

```diff
@@ -37,5 +46,7 @@ const styles = StyleSheet.create({
     fontSize: 14,
     lineHeight: 16,
     marginRight: 24,
+    minWidth: 48,
```

Comment:
Needed to fix width/height so shift from text to spinner didnt cause components to move around